modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,148 @@
+* A tag value must contain only uppercase and lowercase alphanumeric characters, hyphens, underscores, periods, at signs, percent signs, equals signs, plusses, colons, commas, asterisks, pound signs, ampersands, parentheses, square braces, curly braces, and spaces (that is, -_.@%=+:,*$&()[]{} and spaces).
+* A tag key and value must begin and end with an alphanumeric character.
+* Tag value is of type scalar and must be one of the pre-defined values for the key.
+* You can configure a maximum of 50 tags.
+* There should be no tag key with the same value as any of the existing tag keys on a resource (inherited from parent resource).
+
+.Post-cluster creation
+
+After creating {product-title} cluster, you can access the list of the labels and tags defined for the GCP resources by referring to the new fields `.status.platformStatus.gcp.resourceLabels` and `.status.platformStatus.gcp.resourceTags` present in the `infrastructures.config.openshift.io/cluster` object.
+
+.Sample `infrastructure` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+ name: cluster
+spec:
+ platformSpec:
+   type: GCP
+status:
+ infrastructureName: <cluster_id>
+ platform: GCP
+ platformStatus:
+   gcp:
+     resourceLabels:
+     - key: <label_key>
+       value: <label_value>
+     resourceTags:
+     - key: <tag_key_short_name>
+       parentID: <OrganizationID/ProjectID>
+       value: <tag_value_short_name>
+   type: GCP
+----
+
+Along with the user-defined labels, resources will also have a label defined by the {product-title}. The format of the {product-title} labels is `kubernetes-io-cluster-<cluster_id>:owned`. The `cluster_id` parameter is the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
+
+User-defined and {product-title} labels can be found on the following GCP resoucres:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+User-defined tags can be found added to the GCP resources, that is, storage bucket.
+
+The following are the limitations for the user-defined tags:
+
+* Tags will not be attached to master compute instances and storage bucket created by `installer`, worker compute instances created by `machine-api-provider-gcp controller`, filestore instance resources created by `gcp-filestore-csi-driver-operator` and compute disk, compute image resources created by `gcp-pd-csi-driver-operator`.
+* Tags are not supported for buckets located in the following regions:
+** `us-east2` 
+** `us-east3`
+* `cluster-image-registry-operator` will not throw any error but skips processing tags when the buckets are created in the tags unsupported region.
+* Tags must not be restricted to particular service accounts, since operators create and use service accounts with minimal roles.
```

Comment:
```suggestion
* Tags must not be restricted to particular service accounts, because operators create and use service accounts with minimal roles.
```

Reply:
Ack.